src/services/google-sheets.js

```diff
@@ -0,0 +1,118 @@
+import constants from "./constants.js";
```

Comment:
this should go to /services not /src/services

Reply:
also, if we're going to import exactly what we do in radar, we should probably ask @juanigaray to export it from common code.

Reply:
ok